dashboard/lib/services/lti/account_linker.rb

```diff
@@ -14,17 +14,31 @@ def call
         ActiveRecord::Base.transaction do
           user.authentication_options << rehydrated_user.authentication_options.first
           Services::Lti.create_lti_user_identity(user)
+          @lti_integration = user.lti_user_identities.first.lti_integration
           handle_sections(rehydrated_user, user)
           user.lti_roster_sync_enabled = true if user.teacher?
           user.lms_landing_opted_out = true
           PartialRegistration.delete(session)
+          unless rehydrated_user.id
+            # For fresh LTI users who are linking their account, we want to count
+            # them as "new" users for metrics, since they won't hit the registration
+            # controller like they would if they chose to create a new account.
+            Metrics::Events.log_event(
+              user: user,
+              event_name: 'lti_user_created',
+              metadata: {
+                'user_type' => user.user_type,
```

Comment:
Minor FYI, the Statsig wrapper [includes](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/metrics/events.rb#L85) the `user.user_type` in the `custom_ids` hash automatically, so you don't need to include it in the metadata if you don't want to.